docs/reference/migration/migrate_8_1.asciidoc

```diff
@@ -98,3 +98,24 @@ with the correct key.
 ====
 // end::notable-breaking-changes[]
 
+[discrete]
+[[deprecations_81_rest_api]]
+==== REST API deprecations
+
+[[deprecate_index_include_frozen_request_parameter_in_sql_api]]
+.Deprecate `index_include_frozen` request parameter in `_sql` API
+[%collapsible]
+====
+*Details* +
+Following the deprecation of frozen indices, the `index_include_frozen`
+parameter and `FROZEN` syntax is now also deprecated.
+
+*Impact* +
+You should unfreeze frozen indices using the
+{ref}/unfreeze-index-api.html[unfreeze index API] and stop using the
+`index_include_frozen` parameter or the `FROZEN` keyword in SQL
+queries. For some use cases, the frozen tier may be a suitable
+replacement for frozen indices. See {ref}/data-tiers.html[data tiers]
+for more information.
+====
```

Comment:
It looks like `./gradlew generateReleaseNotes` appended new `REST API deprecations` instead of reusing existing one. Fixing this manually for now

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+pull-request+build-docs/33925/console